templates/CRM/Case/Form/CaseView.tpl

```diff
@@ -60,7 +60,7 @@
           <table class="form-layout-compressed">
             {foreach from=$caseRoles.client item=client}
               <tr class="crm-case-caseview-display_name">
-                <td class="label-left bold" style="padding: 0px; border: none;">{$client.display_name}</td>
+                <a href="{crmURL p='civicrm/contact/view' q="action=view&reset=1&cid=`$client.contact_id`"}" title="{ts}View contact record{/ts}">{$client.display_name}</a>
```

Comment:
I don't think this is valid html to have a `<tr>` without a `<td>` inside it.

Reply:
No... I didn't mean to remove the `<td>` - now fixed.